### ISSUE TITLE
Default chain ID

### ISSUE THREAD

**Issue by kauri-hero** [MASQ-Project/MASQ-Node-issues#480]
Inspect and modify the default chain if no value is configured for the chain in setup

@masqrauder  - currently default value is 3

**Comment by bertllll** [comment]
@kauri-hero Will you be satisfied if I preserve the default to be ETH mainnet or do you want a different default?

**Comment by kauri-hero** [comment]
@bertllll I would like the default to be eventually polygon, but since there are a number of cards in flight before that is possible, it might be best to keep it as ETH.

What are your thoughts in terms of the other work in parallel?

We could always make another card that does the final switch of default chains for mainnet and testnet to Polygon and Mumbai
